client/my-sites/domains/domain-management/components/domain/domain-info-card/email/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTranslate } from 'i18n-calypso';
+import Count from 'calypso/components/count';
+import { useEmailAccountsQuery } from 'calypso/data/emails/use-emails-query';
+import { getEmailAddress } from 'calypso/lib/emails';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import DomainInfoCard from '..';
+import { DomainInfoCardProps } from '../types';
+import { EmailAccount } from './types';
+
+const DomainEmailInfoCard = ( { domain, selectedSite }: DomainInfoCardProps ): JSX.Element => {
+	const translate = useTranslate();
+	let emailAddresses: string[] = [];
+
+	const { data, error, isLoading } = useEmailAccountsQuery( selectedSite.ID, domain.name );
+
+	if ( ! isLoading && ! error ) {
+		const emailAccounts: EmailAccount[] = data?.accounts;
+
+		if ( emailAccounts.length ) {
+			emailAddresses = emailAccounts
+				.map( ( a ) => a.emails )
+				.flat()
+				.map( getEmailAddress )
+				.filter( ( email ) => email );
+		}
+	}
+
+	return ! emailAddresses.length ? (
+		<DomainInfoCard
+			type="href"
+			href={ emailManagement( selectedSite.slug, domain.name ) }
+			title={ translate( 'Email' ) }
+			description={ translate( 'Send and receive emails from youremail@%(domainName)s', {
+				args: { domainName: domain.name },
+			} ) }
+			ctaText={ translate( 'Add professional email' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Add Professional Email' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feature/update-domain-management-columns)